conan/api/subapi/new.py

```diff
@@ -36,8 +38,11 @@ def get_builtin_template(template_name):
                          "meson_exe": meson_exe_files,
                          "msbuild_lib": msbuild_lib_files,
                          "msbuild_exe": msbuild_exe_files,
+                         # TODO: Rename xxx_7 to xxx when dropped Bazel 6.x compatibility
                          "bazel_lib": bazel_lib_files,
                          "bazel_exe": bazel_exe_files,
+                         "bazel_lib_7": bazel_lib_files_7,
+                         "bazel_exe_7": bazel_exe_files_7,
```

Comment:
What about changing these names to `bazel_7_lib/bazel_7_exe` or `bazel7_lib/bazel7_exe`?

Reply:
How about removing the version number? Presumably we won't change these attributes for bazel 8, which has the same module API as bazel 7